client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,10 +97,21 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			const moment = this.props.moment;
+			const now = moment();
+			const scheduledDate = moment( this.props.post.date );
+			// If the content is scheduled to be release within a year, do not display the year at the end
+			const displayDate = scheduledDate.diff( now, 'years' ) > 0 ? 'll' : 'D MMM';
```

Comment:
`D MMM` assumes that languages use a day-month format but some use month-day. Couldn't we rather replace the `Y` from the generated `ll`? Something like this:
```
if ( scheduledDate.diff( now, 'years' ) < 1 ) scheduledTime.replace( scheduledDate.format( 'Y' ), '' );
```

Reply:
@mmtr could we please get this addressed also? Thanks!

Reply:
Thanks, fixed! :)